document_merge_service/settings.py

```diff
@@ -300,3 +300,9 @@ def parse_admins(admins):
 
 # App specific arguments for the extension classes
 EXTENSIONS_ARGUMENTS = env.dict("EXTENSIONS_ARGUMENTS", default={})
+
+# By default the maximum number of fields for the data upload is set to 1000:
+# https://docs.djangoproject.com/en/5.1/ref/settings/#data-upload-max-number-fields
+# Since we send about 1500 fields in the post request when creating a new template,
+# we run into a TooManyFiles error. Because of this error, we raise the files to 2000.
+DATA_UPLOAD_MAX_NUMBER_FIELDS = 2000
```

Comment:
Note we're in a public place here - can't make implicit assumptions or statements.

Also, I'd suggest turning this into an env-configurable setting instead, so we won't have to update DMS in the future if this needs changing


```suggestion
# DMS potentially uses a large number of fields when creating templates due to
# the possibility to validate a template against available placeholders.
# https://docs.djangoproject.com/en/5.1/ref/settings/#data-upload-max-number-fields

DATA_UPLOAD_MAX_NUMBER_FIELDS = env.int("DATA_UPLOAD_MAX_NUMBER_FIELDS", default=1000)
```

Reply:
Ah right I didn't consider that. :sweat_smile: Thanks for the hint I changed it.